vllm/v1/metrics/loggers.py

```diff
@@ -137,201 +138,276 @@ def log_engine_initialized(self):
             self.vllm_config.cache_config.num_gpu_blocks)
 
 
-class PrometheusStatLogger(StatLoggerBase):
+class PrometheusMetrics:
+    """
+    vLLM uses a multiprocessing-based frontend for the OpenAI server.
```

Comment:
This isn't true in V0 (at least currently) - it is under consideration to be used for #17546